R/chromote.R

```diff
@@ -436,6 +436,12 @@ Chromote <- R6Class(
     stop = function() {
       private$ws$close()
       private$browser$close()
+      private$browser$get_process()$wait(timeout = 10)
+
+      if (private$browser$is_alive()) {
+        # Still alive after 10 seconds...
+        private$browser$get_process()$kill()
+      }
```

Comment:
```suggestion
      if (private$browser$is_local()) {
        private$browser$get_process()$wait(timeout = 10)

        if (private$browser$is_alive()) {
          # Still alive after 10 seconds...
          private$browser$get_process()$kill()
        }
      }
```

Reply:
Move to `Browser$close()` and add a `wait = FALSE` argument, that also takes an integer seconds to wait or `TRUE` for `10`.

Reply:
Also `process$wait()` throws an error, so kill the process in the error handler

Reply:
Actually, `$stop()` should call `self$close()` if non-local, otherwise tear down local process